tests/sentry/api/endpoints/relocations/test_abort.py

```diff
@@ -13,15 +15,15 @@
 @region_silo_test
 class AbortRelocationTest(APITestCase):
     endpoint = "sentry-api-0-relocations-abort"
+    method = "put"
 
     def setUp(self):
         super().setUp()
         self.owner = self.create_user(
             email="owner", is_superuser=False, is_staff=True, is_active=True
         )
-        self.superuser = self.create_user(
-            "superuser", is_superuser=True, is_staff=True, is_active=True
-        )
+        self.superuser = self.create_user(is_superuser=True)
+        self.staff_user = self.create_user(is_staff=True)
```

Comment:
what happens if a user is both staff and superuser? haha

Reply:
That's actually true for the default `self.user`. It doesn't matter unless you login with both like:
`self.login_as(self.user, superuser=True, staff=True)`. The feature flagged permission class works as expected tho and still only checks staff.